scripts/flow_track_correl.py

```diff
@@ -0,0 +1,414 @@
+import numpy as np
+from astropy.io import fits
+import glob
+import matplotlib
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt
+import os
+
+
+def read_fits(filepath: str, hdu: int = 1) -> np.ndarray:
+    """
+    Read FITS image array.
+
+    Parameters
+    ----------
+    filepath : str
+        Path to the FITS file.
+    hdu : int, optional
+        Index of the header data unit storing the data array, by default 1.
+
+    Returns
+    -------
+    np.ndarray
+        2D image array from the specified FITS file.
+    """
+    with fits.open(filepath) as hdul:
+        hdul.verify('fix')
+        data = hdul[hdu].data
+    return data
+
+
+def read_header(filepath: str, hdu: int = 1) -> fits.header.Header:
```

Comment:
I might suggest reading the data and the header at the same time instead of opening the file twice. Very minor optimization.

Reply:
Had that in mind too. I'll refactor.